packages/byline-parser/index.js

```diff
@@ -34,7 +34,8 @@ export default function (blobSource, { openingPattern, closingPattern, matchPatt
         throw new Error('No capture group is defined');
       }
 
-      tmp.push(captureGroup.slice(1));
+      if (!noGroup) tmp.push(captureGroup.slice(1));
+      if (noGroup) results.push(captureGroup.slice(1));
```

Comment:
I'm not happy with this solution. I tried to avoid code duplication, but now I've the feeling this will be horrible to maintain.